doc/sphinx/source/esmvaldiag/observations.rst

```diff
@@ -0,0 +1,283 @@
+Contributing a CMORizing script for an observational dataset
+************************************************************
+
+ESMValTool is able to process observational data sets, however, since many
+features of this version are based on the `Iris library 
+<https://scitools.org.uk/iris/docs/latest/>`_ that uses the `CF conventions
+<http://cfconventions.org/>`_, the format of the data sets has to follow very
+strict CMOR rules. If the original observational data set is not in the right
+CMOR format, it has to be CMORized before the ESMValTool preprocessor can
+work with it. The following steps are necessary to prepare an observational
+data set for the use in ESMValTool.
+
+| `1. Check if your variable is CMOR standard`_
+| `2. Configuration file`_
+| `3. Store your dataset in the right place`_
+| `4. Reformat your data set to meet CMOR standard - script`_
+| `5. Run the cmorizing script`_
+| `6. Naming convention of the observational data files`_
+| `7. Running a preprocessing test recipe`_
+
+
+1. Check if your variable is CMOR standard
+==========================================
+
+Some variables are defined as CMOR standard. You can find several files with
+CMOR tables in the folder ``/esmvalcore/cmor/tables/cmip5/Tables/``,
+differentiated according to the ``MIP`` they belong to. If you find the
+variable in one of these files, you do not have to do anything further. 
+
+If your variable is somehow more specific than the entries in the tables
+(e.g. your variable is called ``bihemispherical directional albedo`` and in
+the table only ``albedo`` is available), please write your own custom
+variable information as outlined below. If you do not find your variable
+here, you have to provide the variable information in an additional file
+that will have to be placed in he folder ``/esmvalcore/cmor/tables/custom/``.
+
+The file that you have to create for a custom variable needs to follow these
+guidelines:
+
+- Provide information about the SOURCE (for all observations and reanalysis it
+  was decided by the ESMValTool development team that this is set to ``CMIP5``;
+  it basically describes the framework in which the individual variables are
+  associated with realms; it might not be necessary to provide this information
+  here, but before we know for sure that we can leave it out, please just
+  provide it);
+- Provide the ``variable_entry``;
+- Provide the ``modeling_realm``;
+- Provide the variable attributes, but **do not** provide a standard name! The
+  custom variables will only be read by Iris if this information is not
+  provided! Necessary variable attributes are: ``units``, ``cell_methods``,
+  ``cell_measures``, ``long_name``, ``comment``.  
+- Provide some additional variable attributes. Necessary additional variable
+  attributes are: ``dimensions``, ``out_name``, ``type``. There are also
+  additional variable attributes that can be defined here (see the already
+  available cmorizers). 
```

Comment:
Which additional attributes?

Reply:
I meant here additional attributes listed in the paragraph (e.g. "dimensions").